src/awkward/_connect/rdataframe/from_rdataframe.py

```diff
@@ -130,6 +130,10 @@ def form_dtype(form):
     column_types = {}
     result_ptrs = {}
     contents = {}
+    awkward_type_cols = {}
+
+    columns = columns + ("rdfentry_",)
```

Comment:
Looking at ROOT documentation: an `rdfentry_` column is an alias or a replacement of a `tdfentry_` column that was introduced in versions greater than 6.14. The latter legacy column name is still supported.